### DISCUSSION TITLE
Wrote an article on how to create a simple ChatGPT clone using langchaingo

### DISCUSSION THREAD

**Discussion by sausheong** [tmc/langchaingo#239] [Show and tell]
Thanks for creating a great project! I wrote an article on creating a simple ChatGPT clone using library, works perfectly! Just sharing it here. Will be writing more.

https://sausheong.com/creating-a-simple-chatgpt-clone-with-go-c40b4bec9267?sk=53a2bcf4ce3b0cfae1a4c26897c0deb0

**Comment by tmc** [comment]
Very cool! perhaps I should add a section to the README linking out to resources like this.. (feel free to open a PR to link to it).

**Reply by sausheong** [reply]
Ok! I submitted a PR on this. Added a section in README.md with 2 articles I wrote so far.
